### PR TITLE
[bitnami/kafka] Make it possible to customize the securityContext in all containers

### PR DESCRIPTION
**Pull request by madrisan**
Make it possible to customize the securityContext in all containers

**Description of the change**

Allow the customization of the security context in all the instantiated containers.

**Benefits**

This PR makes it possible to deploy the kafka metric containers in kubernetes environments where some mandatory settings of the securityContext setup are checker before deploying a container.

**Possible drawbacks**

None known because the default settings do not change the chart behaviour.

**Applicable issues**

None

**Additional information**

None

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Comment by madrisan** [comment]
Ping.

File: bitnami/kafka/README.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ The command removes all the Kubernetes components associated with the chart and
...
 | `podSecurityContext.fsGroup`         | Group ID for the filesystem used by the containers                                                                                                                                            | `1001`          |
 | `podSecurityContext.runAsUser`       | User ID for the service user running the pod                                                                                                                                                  | `1001`          |
-| `containerSecurityContext`           | Kafka containers' Security Context                                                                                                                                                            | `{}`            |
+| `containerSecurityContext.enabled`   | Kafka containers' Security Context                                                                                                                                                            | `false`         |
```
Comment (juan131) [inline bitnami/kafka/README.md:181]: This description doesn't match the one you indicated in the values metadata (using `## @param  ... `)
Please use the [readme-generator](https://github.com/bitnami-labs/readme-generator-for-helm) tool to ensure the README.md tables are synced with the values metadata.

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks so much for this contribution, we really appreciate it! Please take a look to my comments when you have a chance.

File: bitnami/kafka/README.md (hunk 1 above)
Comment (madrisan) [reply bitnami/kafka/README.md]: Should be fine now: no errors reported by `readme-generator`.

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**